distsql/select_result.go

```diff
@@ -53,6 +53,11 @@ var (
 	errQueryInterrupted = dbterror.ClassExecutor.NewStd(errno.ErrQueryInterrupted)
 )
 
+var (
+	telemetryStoreBatchedCnt         = metrics.TelemetryStoreBatchedCnt
```

Comment:
It seems it's not enough to see the percentage of batched tasks like `batched_task_cnt/total_task_cnt`?

Reply:
I'll also record the total task for batched cop iterator